mathics/core/element.py

```diff
@@ -145,6 +145,23 @@ class KeyComparable:
     """
 
     def get_sort_key(self):
```

Comment:
When I read a sentence like "should provide a list" I think, hey why not just say that in the code. like:

```
def get_sort_key() -> list: 
```

It appears though that we really man something more. It is in fact a tuple (an immutable list of a fixed number of items where the types have to be particular things).

For now, just going this little bit further would be appreciated. 

But some point in the future, I hope we go further and get more precise about what a `get_sort_key()` returns.

Reply:
Indeed, it should be a `tuple`. But for some reason, a `list` was chosen. When this be ready, we can change the type in a different PR.